src/encoded/search_views.py

```diff
@@ -171,29 +173,22 @@ def matrixv2_raw(context, request):
     return fr.render()
 
 
-@view_config(route_name='matrix', request_method='GET', permission='search')
-def matrix(context, request):
+def generic_matrix(request, title_response_field_params=None, type_response_field_params=None, response_field_params=None):
```

Comment:
Let's actually not do the generic_matrix thing here. It will be cleaner just to make your own target_matrix view.

Reply:
Done.